src/main/java/app/gpx_animator/Renderer.java

```diff
@@ -256,17 +287,12 @@ private void parseGPX(final List<Long[]> spanList, final TreeMap<Long, Point2D>
         for (final TrackConfiguration trackConfiguration : cfg.getTrackConfigurationList()) {
             trackIndex++;
 
-            final File inputGpxFile = trackConfiguration.getInputGpx();
             final GpxContentHandler gch = new GpxContentHandler();
-            GpxParser.parseGpx(inputGpxFile, gch);
+            GpxParser.parseGpx(trackConfiguration.getInputGpx(), gch);
 
             final List<TreeMap<Long, Point2D>> timePointMapList = new ArrayList<>();
 
-            final List<List<LatLon>> pointLists = gch.getPointLists();
-            if (pointLists.isEmpty()) {
-                throw new UserException(resourceBundle.getString("renderer.error.notrack").formatted(inputGpxFile));
-            }
-            for (final List<LatLon> latLonList : pointLists) {
+            for (final List<LatLon> latLonList : gch.getPointLists()) {
```

Comment:
I added this check and error message a couple of days ago. A pull from upstream should fix it, too.